code/modules/atmospherics/machinery/components/unary_devices/tank.dm

```diff
@@ -9,6 +9,7 @@
 	density = TRUE
 	var/gas_type = 0
 	layer = ABOVE_WINDOW_LAYER
+	pipe_flags = PIPING_ONE_PER_TURF
```

Comment:
can you apply this to all unary devices?

Reply:
Kk

Reply:
Gimme a bit

Reply:
Apply to trinary too, please.

Reply:
it's already applied to trinary